adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/AdyenSummaryCheckoutStepController.java

```diff
@@ -173,6 +174,11 @@ public String placeOrder(@ModelAttribute("placeOrderForm") final PlaceOrderForm
                 if (PAYMENT_METHOD_BOLETO.equals(cartData.getAdyenPaymentMethod())) {
                     addBoletoMessage(redirectModel, orderData.getCode());
                 }
+
+                if (PAYMENT_METHOD_MULTIBANCO.equals(cartData.getAdyenPaymentMethod())) {
```

Comment:
Please add else here to prevent if condition to be evaluated if payment method is boleto.